codecademy/day_at_the_supermarket.py

```diff
@@ -0,0 +1,29 @@
+# from collections import Counter
+
+stock = {
+    "banana": 6,
+    "apple": 0,
+    "orange": 32,
+    "pear": 15
+}
+
+prices = {
+    "banana": 4,
+    "apple": 2,
+    "orange": 1.5,
+    "pear": 3
+}
+
+
+# Write your code below!
+def compute_bill(food):
+    assert isinstance(food, (list, tuple, set)), "{} error enter type".format(food)
+    total = 0
+    for item in food:
+        assert item in stock, "{} error enter type".format(food)
```

Comment:
too restrictive
here is a use case:
customer wants to buy potatoes and bananas, but you only have bananas in stock, so assert will through exception on potatoes and the sale will not go through, meaning the function will not return anything but interrupt and through exception
so i'd print("error this item is not in stock") instead and proceed with the rest of the items